src/agent/debuglet.ts

```diff
@@ -62,6 +62,13 @@ const BREAKPOINT_ACTION_MESSAGE =
     'The only currently supported breakpoint actions' +
     ' are CAPTURE and LOG.';
 
+// PROMISE_RESOLVE_CUT_OFF_IN_MILLISECONDS is a heuristic that we set to force
+// the debug agent to return a promise when isReady is called in
+// isReadyManager. The value is selected between Stackdriver debugger hanging
+// get duration (40s) and TCP default time-out
+// (https://tools.ietf.org/html/rfc5482).
+const PROMISE_RESOLVE_CUT_OFF_IN_MILLISECONDS = 60 * 1000;
```

Comment:
Why is this 60 seconds and not 100 seconds. The comment should explain why this number is reasonable. It mentions that it should be somewhere in the range.. but where in the range